### PR TITLE
improvement: add advanced section to python readme

### PR DESCRIPTION
**Pull request by armandobelardo**

### CODE REVIEW

File: seed/python-sdk/api-wide-base-path/README.md [hunk 1]
```diff
@@ -57,6 +57,69 @@ asyncio.run(
...
+A request is deemed retriable when any of the following HTTP status codes is returned:
+
+- [408](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/408) (Timeout)
+- [429](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429) (Too Many Requests)
+- [5XX](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500) (Internal Server Errors)
+
+Use the `max_retries` request option to configure this behavior.
+
+```python
+client.service.post({
```
Comment (dsinghvi) [inline seed/python-sdk/api-wide-base-path/README.md:90]: nit: im guessing we'll need `...` ?

File: seed/python-sdk/api-wide-base-path/README.md [hunk 2]
```diff
@@ -57,6 +57,69 @@ asyncio.run(
...
+
+## Custom Client
+
+You can override the `httpx` client to customize it for your use-case. Some common use-cases include support for proxies
+and transports.
+```python
+import httpx
+from seed.client import SeedApiWideBasePath
+
+client = SeedApiWideBasePath(..., http_client=httpx.Client(proxies=http://my.test.proxy.example.com, transport=httpx.HTTPTransport(local_address="0.0.0.0"), ), )
```
Comment (dsinghvi) [inline seed/python-sdk/api-wide-base-path/README.md:120]: weird that this is how it gets formatted -- is this affected by the trailing comma?

Comment (armandobelardo) [reply seed/python-sdk/api-wide-base-path/README.md:120]: Right? I'm actually not sure why it gets formatted this way, I need to see what's happening here

Comment (armandobelardo) [reply seed/python-sdk/api-wide-base-path/README.md:120]: Fixed!

**Review by dsinghvi — APPROVED** [review]
🚀

**Pull request merged**